rpc_policy.go

```diff
@@ -19,6 +19,7 @@ var DefaultRPCPolicy = map[string]RPCEndpointType{
 	"Cluster.PinPath":            RPCClosed,
 	"Cluster.Pins":               RPCClosed, // Used in stateless tracker, ipfsproxy, restapi
 	"Cluster.Recover":            RPCClosed,
+	"Cluster.RecoverAll":         RPCClosed,
```

Comment:
`RecoverAllLocal` and `RecoverLocal` should be RPCTrusted though (with comment `// Called in broadcast from RecoverAll() / Recover()`)